convert/convert_gptoss.go

```diff
@@ -110,9 +110,12 @@ func (m *gptossModel) Tensors(ts []Tensor) []*ggml.Tensor {
 
 	for name, mxfp4 := range mxfp4s {
 		dims := mxfp4.blocks.Shape()
+		if !strings.HasSuffix(name, ".weight") {
```

Comment:
this is an unrelated change where the original gpt oss model already ends the tensor with `.weight` so unconditionally adding `.weight` creates a tensor ending in `.weight.weight`